src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static nature() {
+    this.alive = this.alive.filter((dead) =>
+      dead.health !== 0);
```

Comment:
The method 'nature' should be renamed to better reflect its purpose. It seems to be responsible for removing dead animals from the 'alive' array, so a name like 'removeDead' would be more appropriate and descriptive.